cmd/noms/noms_log.go

```diff
@@ -99,8 +100,12 @@ func runLog(args []string) int {
 	defer pgr.Stop()
 
 	for commitBuff := range outChan {
-		io.Copy(pgr.Writer, bytes.NewReader(commitBuff.([]byte)))
+		_, err := io.Copy(pgr.Writer, bytes.NewReader(commitBuff.([]byte)))
+		if err != nil {
+			os.Exit(0)
```

Comment:
I've had code like this in the past and it caused other problems, like the terminal not resetting correctly. Any idea what is still running? Does just `break` here fix it (in which case I would wonder why)?